wiki/Legal/Privacy/ru.md

```diff
@@ -1,6 +1,7 @@
 ---
 legal: true
 outdated: true
+outdated_since: e95d1bf509dcb730b46f93ef70215e84f57bc37f
```

Comment:
uhh no, if it's already outdated, then you don't know how many commits behind it is, and the diff may be (and will be) misleading. same goes for `zh.md`